app/views/orgs/roster_entries/assignment_repos/_linked_not_accepted.html.erb

```diff
@@ -8,7 +8,11 @@
     <div class='d-flex flex-column flex-justify-around'>
       <div>
         <%= link_to current_roster_entry.user.github_user.html_url, class: "assignment-repo-github-url", target: '_blank' do %>
-          <h3 class="css-truncate css-truncate-target"><%= current_roster_entry.user.github_user.login %></h3>
+          <% if current_roster_entry.identifier %>
```

Comment:
Are there any roster entries without identifiers? Or can we :fire: this line? This view partial is specific to students linked roster entries, so I _think_ we're good.